Tools/Scripts/webkitpy/layout_tests/models/test_run_results_unittest.py

```diff
@@ -158,6 +159,7 @@ def test_no_git_revision(self):
         summary = summarized_results(self.port, expected=False, passing=False, flaky=False)
         self.assertNotIn('revision', summary)
 
+    @slow
```

Comment:
@JonWBedard my one question here is whether you think this one test has any value, or whether we should just delete it in favour of the (near identical) one below that uses a mock

Reply:
I would be in favor of deleting. Our mock Git foo is very heavily tested, I don't see a benefit of using actual git here.

r+ing in it's current status, r+ stands if this test is removed (which would be my preference, but not strong enough to block landing as is)